acq4/util/clibrary/CParser.py

```diff
@@ -482,7 +483,7 @@ def pa(t):
                 elif d == 'pragma':  ## Check for changes in structure packing
                     if not ifTrue[-1]:
                         continue
-                    m = re.match(r'\s+pack\s*\(([^\)]+)\)', rest)
+                    m = re.match(r'\s+pack\s*\(([^)]+)\)', rest)
```

Comment:
why?

Reply:
Have you seen the rent on a backslash? Those things are easily 20% more expensive than other characters.

(Really, this was part of the "hey pycharm, are there any egregious errors in this file?" check, and it included this unnecessary escape in said category and I said "sure, buddy; if you say so." and didn't give it any more thought.)

Reply:
Oh oh I see, the escape is unnecessary inside `[]`